_data/master/navbars/getting-started.yml

```diff
@@ -46,6 +46,11 @@ toc:
   - title: Upgrade
     path: /getting-started/kubernetes/upgrade
 
+- title: OpenShift
+  section:
+    - title: Installation
+      path: /getting-started/openshift/installation
```

Comment:
Why add this as its own section?
Is OpenShift that different than straight up k8s that it should be on its own?
If it should remain on its own then I think at a minimum there should be an 'Overview' page added.
One question that may be a good judge of where it should go is, do our Tutorials in the K8s section apply to OpenShift also?